WooCommerce/Classes/ViewRelated/CardPresentPayments/CardPresentPaymentsModalViewController.swift

```diff
@@ -277,14 +279,29 @@ private extension CardPresentPaymentsModalViewController {
     }
 
     func configureAuxiliaryButton() {
+        clearAuxiliaryButton()
+
         guard shouldShowAuxiliaryButton() else {
             auxiliaryButton.isHidden = true
             return
         }
 
-        auxiliaryButton.setTitleWithoutAnimation(viewModel.auxiliaryButtonTitle, for: .normal)
         auxiliaryButton.isHidden = false
+        auxiliaryButton.setTitle(viewModel.auxiliaryButtonTitle, for: .normal)
         auxiliaryButton.accessibilityIdentifier = Accessibility.auxiliaryButton
+
+        if viewModel.actionsMode == .secondaryActionAndAttributedAuxiliaryButton {
+            auxiliaryButton.setImage(.infoOutlineImage, for: .normal)
+            auxiliaryButton.setAttributedTitle(viewModel.auxiliaryAttributedButtonTitle, for: .normal)
+            auxiliaryButton.distributeTitleAndImage(spacing: 12.0)
+        }
+    }
+
+    func clearAuxiliaryButton() {
+        auxiliaryButton.setImage(nil, for: .normal)
+        auxiliaryButton.setAttributedTitle(nil, for: .normal)
+        auxiliaryButton.setTitle(nil, for: .normal)
+        auxiliaryButton.accessibilityIdentifier = nil
```

Comment:
This method _may_ become unnecessary as well, if you do the above... because things that should be nil will be set to nil based on the viewModel property.